_posts/2020-06-21-covid-decision.md

```diff
@@ -0,0 +1,134 @@
+author: Jessica White
+categories: [dddem]
+comments: false
+share: true
+---
+
+----
+<center>
+<h3 class="quote"><i>Our Final Decision And How We Reached It</i> </h3>
+</center>
+
+---
+<br/>
+
+The DDD East Midlands team have decided that the most responsible thing to do is **postpone the DDD East Midlands Conference until 2021**. Though this has been a tough decision to make, the safety of our delegates is of the upmost importance.
+
+<br/>
+
+# Why we came to this decision
+
+The team met every 4 weeks to assess the statistics around COVID-19 and what it would take to run a safe and successful conference. As two of our team are currently shielding, this also included the ability or organise the conference effectively.
+
+It looks likely that some form of social distancing will still be in place by the time October comes around. Not only is the overhead of ensuring social distancing is adhered to a logistical challenge, but we think it will have a big effect on the feel of the conference itself. 
+
+DDD East Midlands is a chance for the community and local businesses to celebrate and showcase all that makes the local tech community great. There are a range of activities planned for the conference and a lot of this includes some range of closeness or contact with surfaces. 
+
+The health and safety of our delegates, and that they have a fantastic experience of the event is really important to us.
+
+### The other options we considered.
+
+Like many conferences and events this year, we did consider moving to a virtual event. We have decided not to do this for a number of reasons. Firstly, we want to showcase the tech and talent of the East Midlands, which is best done in location. Secondly, there are many online events at the moment which are offering great content which we think meet this need well already. Thirdly, it would be difficult to recreate the same atmosphere and experience as the in person event. 
+
+<br/>
+<img src="/assets/img/2020-06-21/flags.jpg" alt="The DDD Flags outside Nottingham Conference Centre" width="100%"/>
+<center>
+<i>The DDD Flags outside Nottingham Conference Centre</i>
+</center>
+<br/>
+
+---
+<br/>
+
+# Common questions 
+
+We understand there will be a lot of questions around this. Hopefully, the below will answer a few of them. As always, we are here to answer any others you may have.
+
+### What day is DDD East Midlands 2021?
+
+Saturday 2nd October 2021.
+
+### Can I carry over the ticket I have for 2020?
+
+Yes. We will be closing off the ability to sign up for a ticket until next year, but any non-cancelled tickets will be carried over to 2021. 
+
+### Will you be pausing ticket sales.
+
+Yes. Though tickets are being carried over, we will close off tickets for now while we plan for 2021.
+
+### What happens for speakers?
+
+We are still hosting the online virtual workshop for the speakers selected for 2020, but we have decided to re-start the talk submission and voting process for 2021. 
```

Comment:
It reads that submissions for 2021 are open for talks? Could change to "..but we have decided we will re-start the..." so it's clear submissions aren't open yet for 2021